### PR TITLE
Add document about "Code Review"

### PR DESCRIPTION
**Pull request by NVerdie**

### CODE REVIEW

File: code_review.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Code Review
+
+
+![alt text](./imagecodereview.jpg "Code review image")
```
Comment (thibautcornolti) [inline code_review.md]: Please use a resource folder

File: code_review.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+Code review is a review of the source code of a project. The overall goal is to find bugs, potential vulnerabilities, to correct design errors or to have a good readability of the code. All this with the aim of improving the quality, maintainability and security of the project.
+The objectives of the code review are :
+- Improve code quality
+- Encouraging collaboration, teamwork and teamwork
+- Apply a code standard
+- Detecting and correcting faults in each push
+
+### When to do a code review:
+    You can do a code review every push if you feel the need. However, on minor pushs, this can be a waste of time. That's why it's better to do a code review when you add a feature to the project or when the push modifies code already in the project. This will allow you to make sure that the work you are going to share does not jeopardize what has already been done. Of course, a general code review will have to be done at each big step, for example before merging work on the production, pre-production or master branches.
```
Comment (thibautcornolti) [inline code_review.md]: pushes instead of pushs

**Review by thibautcornolti — CHANGES_REQUESTED** [review]
Some minor fixes

**Comment by thibautcornolti** [comment]
Please aim develop instead of master

**Review by thibautcornolti — APPROVED** [review]
Very Good Nicolas !!!

**Comment by bastienrinck** [comment]
Sounds good to me, validating and proceeding with the merge.

**Review by bastienrinck — APPROVED** [review]

**Pull request merged**